engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -13,6 +13,8 @@ object TerminalLayout {
   val Converter = new ThrowableProxyConverter
   Converter.start()
 
+  private val dateTimeFormatter = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss,SS")
```

Comment:
This goes to the terminal, and meant for humans, not machines, afaik. So leaving as ~~is~~ was.